draft-ietf-scitt-architecture.md

```diff
@@ -267,15 +263,9 @@ The Statement is considered opaque to Transparency Service, and MAY be encrypted
 
 Subject:
 
-: This term has the same definition as in {{RFC8392}}, which relies on the definition in RFC7519.
-The `sub` (subject) claim identifies the principal that is the subject of the CWT.
-The claims in a CWT are normally statements about the subject.
-In SCITT, `sub` identifies the entity about which statements, and receipts are made.
-The subject value MUST either be scoped to be locally unique in the context of the Issuer or be globally unique.
-The processing of this claim is generally application specific.
-The `sub` value is a case-sensitive string containing a StringOrURI value.
-Issuers use `sub` to identify the entity about which they are making Signed Statements.
-Transparency Services use `sub` to identify the entity about which they are issuing a Receipt.
+: an identifier that represents the organization, device, user, entity or Artifact about which Statements (and Receipts) are made and by which a logical collection of Statements can be grouped.
```

Comment:
```suggestion
: an identifier that represents the organization, device, user, entity or Artifact about which Statements (and Receipts) are made and by which a logical collection of Statements can be grouped. A Subject refers to whatever the Issuer wants it to refer to. Examples include the Signed Statement or the Statements (payload of the Signed Statement).
```

Reply:
A little bit more colloquial, but maybe it is even better to be silent on this?

Reply:
```suggestion
: an identifier, defined by the Issuer, that represents the organization, device, user, entity, or Artifact about which Statements (and Receipts) are made and by which a logical collection of Statements can be grouped.
```